docs/site/migration/not-planned.md

```diff
@@ -10,8 +10,106 @@ permalink: /doc/en/lb4/migration-not-planned.html
 Missing instructions for your LoopBack 3 use case? Please report a [Migration docs issue](https://github.com/strongloop/loopback-next/issues/new?labels=question,Migration,Docs&template=Migration_docs.md) on GitHub to let us know.
 " %}
 
-{% include note.html content="
-This is a placeholder page, the task of adding content is tracked by the
-following GitHub issue:
-[loopback-next#3956](https://github.com/strongloop/loopback-next/issues/3956)
-" %}
+In the early days of LoopBack, we were exploring different functional areas to
+find features that would make the framework compelling to a wide audience. Some
+of those feature become popular, other not so much. In the meantime, the world
+has moved on and many paradigms popular back in 2013-2015 have been superseded
+by better ideas.
+
+Additionally, the community of LoopBack maintainers was not growing fast enough
+to support the wide spectrum of features offered by LoopBack. As a result, most
+extensions were not integrated well together and many of them are becoming
+outdated by now.
+
+In order to keep the project sustainable and offer great developer experience to
+framework users, we decided to sunset several LoopBack 3 extensions and
+features. We are encouraging our community step up and take over development of
+these extensions.
+
+## Storage component
+
+[loopback-component-storage](https://loopback.io/doc/en/lb3/Storage-component.html)
+provides a unified REST API for storing and retrieving arbitrary files, using
+3rd-party storage like Amazon S3.
+
+Essentially, this component has the following parts:
+
+1. A custom layer to handle file uploads
+2. A custom layer to handle file downloads
+3. Connectors to different storage services (this part is handled by
+   [pkgcloud](https://www.npmjs.com/package/pkgcloud) package)
+4. Integration layer providing File and Container abstractions with REST API.
+
+In LoopBack 4, you can implement a similar functionality as follows:
+
+1. Use `x-parser: stream` extension to receive request body as a stream, and
+   [multer](https://www.npmjs.com/package/multer) package to deal with
+   multi-part file uploads. Check out our
+   [File Upload Example](https://github.com/strongloop/loopback-next/tree/master/examples/file-upload)
+   for a fully working application.
+
+2. LoopBack 4 does not provide first-class support for file downloads yet, see
+   [loopback-next#2230](https://github.com/strongloop/loopback-next/issues/2230).
+   As a workaround, you can inject the full Express response object and use
+   Express API to stream the response body. (See the discussion in the linked
+   GitHub issue for more details.)
```

Comment:
We should mention https://github.com/strongloop/loopback-next/tree/master/examples/file-upload. It's also possible to use https://github.com/dustin-H/multer-storage-pkgcloud as the storage engine for `multer`.

There is a possibility to build a component based on the code from `examples/file-upload`.

Reply:
+1 to mention the file upload example in this second step describing file downloads.

I am not comfortable recommending multer-storage-pkgcloud because it seems to be abandoned for 4 years (the last commit was in Feb 2016).

<img width="1000" alt="Screen Shot 2020-03-13 at 15 56 14" src="https://user-images.githubusercontent.com/1140553/76632290-4d297c00-6543-11ea-878d-909c0640db80.png">

Reply:
Addressed in d6c79f4bd, LGTY now?